src/pages/LandingPage.css

```diff
@@ -45,3 +45,32 @@
 .landing-text .welcome-text {
   margin-bottom: 4%;
 }
+
+@media only screen and (max-width: 570px) {
```

Comment:
Can we adopt landing page for screen sizes between desktop and mobile (for example iPad) so that login button text do not go outside the button?